openapi-cli/src/main/resources/cli-help/ballerina-openapi-sanitize.help

```diff
@@ -12,9 +12,7 @@ SYNOPSIS
 
 DESCRIPTION
        Sanitize the OpenAPI contract file according to the best naming
-       practices of Ballerina. The OpenAPI contract is flatten and the type
-       schema names are made Ballerina friendly. The Ballerina name
-       extensions are added to the schemas which can not be modified
+       practices of Ballerina. The Ballerina name extensions are added to the schemas which can not be modified
```

Comment:
There is a character limit when we print it in the console can we split it to the next line
```help
DESCRIPTION
       Sanitize the OpenAPI contract file according to the best naming
       practices of Ballerina. The Ballerina name extensions are added
       to the schemas which can not be modified directly.
```